media/js/mozorg/home/home.js

```diff
@@ -2,147 +2,203 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-(function(Waypoint) {
-    'use strict';
-
-    function getNextEl(el) {
-        el = el.nextSibling;
-        while (el) {
-            if (el.nodeType === 1) {
-                return el;
-            }
-            el = el.nextSibling;
-        }
-        return null;
-    }
+// YouTube API hook has to be in global scope
+ function onYouTubeIframeAPIReady() {
+     'use strict';
+
+     Mozilla.firstRunOnYouTubeIframeAPIReady();
+ }
+
+ (function(Waypoint) {
+     'use strict';
+
+     var tag = document.createElement('script');
+     tag.src = 'https://www.youtube.com/iframe_api';
+     var firstScriptTag = document.getElementsByTagName('script')[0];
+     firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
+    //
+     function onYouTubeIframeAPIReady() {
+
+        // lazy load video when visitor clicks the placeholder.
+         var videoLink = document.querySelector('.video-play');
+         var videoId = videoLink.getAttribute('data-id');
+
+         new YT.Player(videoLink, {
```

Comment:
Hmm, right now this is trying to initialize on page load, which triggers the video autoplay blocking feature in Nightly:

<img width="479" alt="screen shot 2018-09-10 at 10 00 01" src="https://user-images.githubusercontent.com/400117/45288243-1d2e9200-b4e2-11e8-941d-41580bf5aef2.png">

What we want to do is only load the video player when someone clicks a vide card to open the modal, so inside here we likely want to bind the event listener for the card click events, similar to how we do it for the [berlin page](https://github.com/mozilla/bedrock/blob/master/media/js/firefox/new/berlin-scene1.js#L31).